### DISCUSSION TITLE
Add optional input with regexp expressions separated by new lines that can also trigger a bump

### DISCUSSION THREAD

**Discussion by eversardoth** [PaulHatch/semantic-version#103] [Ideas]
Hello,

I would like to contribute to the project by implementing this feature. I'm starting to use this action, and I really like all the features it provides. Nevertheless, this feature can provide a more robust versioning system (in case of GitHub Actions API downtime when merging) when there is development in multiple topic branches at the same time.

Thanks,

Ever.

**Comment by PaulHatch** [comment]
Hi @eversardoth, regex already supports multiple matching expressions [using `|` delineation as a logical OR](https://en.wikibooks.org/wiki/Regular_Expressions/operator/pipe). I am not sure exactly what you mean by the Actions API being down, but I think this should be all that's needed to provide multiple matching patterns.
